srcpkgs/leanify/template

```diff
@@ -0,0 +1,17 @@
+# Template file for 'leanify'
+pkgname=leanify
```

Comment:
Package name should be `Leanify`, we try to preserve upstream case when possible